packages/ui/src/components/navigation/multi-step-cards/next-button/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { ReactElement } from "react";
+import { ReactComponent as ChevronDown } from "../../../../assets/chevron-down.svg";
+import { Button, ButtonProps } from "../../../input";
+
+export type NextButtonProps = ButtonProps;
+
+export const NextStepButton = ({
+    children,
+    ...rest
+}: NextButtonProps): ReactElement => {
+    return (
+        <div className="cui-absolute cui-bottom-0 cui-left-1/2 cui-transform -cui-translate-x-1/2 cui-translate-y-1/2">
+            <Button
+                {...rest}
+                size="small"
+                className={{
+                    // FIXME: find out why important is needed, fix
```

Comment:
About this we may need to look into this I think https://github.com/dcastil/tailwind-merge, but we'll see.

Reply:
Good suggestion; added [this](https://github.com/carrot-kpi/v1-monorepo/issues/124) issue to make us remember